Packs/AnsibleTower/Playbooks/Ansible_Tower-_job_generic_polling.yml

```diff
@@ -237,48 +244,85 @@ tasks:
       print_output:
         complex:
           root: inputs.show_output
+      text_filter:
+        simple: RECAP
     separatecontext: false
     view: |-
       {
         "position": {
-          "x": 630,
-          "y": 990
+          "x": 162.5,
+          "y": 720
         }
       }
     note: false
     timertriggers: []
     ignoreworker: false
     skipunavailable: false
     quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "7":
+    id: "7"
+    taskid: bd418f5e-49e2-43d0-8c1e-84381ce4773d
+    type: regular
+    task:
+      id: bd418f5e-49e2-43d0-8c1e-84381ce4773d
+      version: -1
+      name: Get Job Events
+      description: Retrieves the list of job events of specific job.
+      script: '|||ansible-tower-job-events-list-by-id'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "3"
+    scriptarguments:
+      job_id:
+        simple: ${AnsibleAWX.Job.id}
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 162.5,
+          "y": 895
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
 view: |-
   {
     "linkLabelsPosition": {},
     "paper": {
       "dimensions": {
-        "height": 1405,
-        "width": 740,
-        "x": 270,
+        "height": 1085,
+        "width": 492.5,
+        "x": 50,
         "y": 50
       }
     }
   }
 inputs:
```

Comment:
@ayman-m As discussed, please add an input to choose whether to run the new task or not.